torchaudio/csrc/sox_effects.h

```diff
@@ -18,8 +18,8 @@ c10::intrusive_ptr<torchaudio::sox_utils::TensorSignal> apply_effects_tensor(
 c10::intrusive_ptr<torchaudio::sox_utils::TensorSignal> apply_effects_file(
     const std::string path,
     std::vector<std::vector<std::string>> effects,
-    const bool normalize = true,
-    const bool channels_first = true);
+    c10::optional<bool>& normalize,
```

Comment:
I don't think passing by reference here actually makes a difference

Reply:
I read these and I thought it might be, so I took the safer option.

https://fb.workplace.com/groups/pytorch.dev/permalink/768545893723893/
https://fb.workplace.com/groups/pytorch.dev/permalink/717862835458866/